app/src/main/java/com/udacity/nanodegree/blooddonation/ui/home/view/HomeActivity.java

```diff
@@ -15,98 +18,179 @@
 import com.google.android.gms.maps.SupportMapFragment;
 import com.google.android.gms.maps.model.BitmapDescriptorFactory;
 import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.Marker;
 import com.google.android.gms.maps.model.MarkerOptions;
 import com.udacity.nanodegree.blooddonation.R;
 import com.udacity.nanodegree.blooddonation.base.BaseActivity;
 import com.udacity.nanodegree.blooddonation.databinding.ActivityHomeBinding;
-import com.udacity.nanodegree.blooddonation.ui.home.presenter.HomeActivityPresenter;
 import com.udacity.nanodegree.blooddonation.ui.home.HomeActivityContract;
+import com.udacity.nanodegree.blooddonation.ui.home.presenter.HomeActivityPresenter;
 
 /**
  * Created by Ankush Grover(ankushgrover02@gmail.com) on 23/04/2018.
  */
-public class HomeActivity extends BaseActivity
-    implements OnMapReadyCallback, HomeActivityContract.View {
+public class HomeActivity extends BaseActivity implements OnMapReadyCallback, HomeActivityContract.View, GoogleMap.OnMarkerClickListener {
+
+    private GoogleMap mMap;
+    private HomeActivityPresenter mPresenter;
+    private LinearLayout mDonorSheet, mReceiver;
+    private BottomSheetBehavior<LinearLayout> donorBehavior;
+    private BottomSheetBehavior<LinearLayout> receiverBehaviour;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        mBinding = DataBindingUtil.setContentView(this, R.layout.activity_home);
+
+        mPresenter = new HomeActivityPresenter(this);
+        mDonorSheet = findViewById(R.id.donor_sheet);
```

Comment:
You can use data binding here, instead of doing findViewById.

Reply:
No over here it is not possible